### ISSUE TITLE
 DDplus, Atmos(if any) Audio

### ISSUE THREAD

**Issue by cavalia88** [jangxx/netflix-1080p#17]
The jangxx/netflix-1080p plug-in currently only provides 5.1 sound

Would it be possible to include option to enable " DDplus, Atmos(if any) Audio"? Something similar to https://github.com/lkmvip/netflix-4K-DDplus (which unfortunately only works on Edge chromium)

**Comment by jangxx** [comment]
Unfortunately not, since Chrome can not decrypt those streams as far as I know. Maybe @lkmvip can chime in and correct me if I'm wrong, but I'm pretty sure that this is not possible in Chrome.

**Comment by msaifuddin** [comment]
The audio stream is not encrypted, only the video. However, Dolby Audio is not supported on Chrome.

[Dolby Digital Plus is available today in **Microsoft Edge** (Windows 10 and above), Apple Safari 9 on OS X 10.11 (El Capitan and above), and Apple Safari 9 on iOS 9.3 and above.](https://developer.dolby.com/platforms/html5/overview/)

**Comment by fis60** [comment]
Am sure that Netflix Audio is not encrypted at all and you can download it to test, but i don’t know how the request is made
to get it, as for avc main video this can only work on playready drm so if you turn off playready on egde you can still get 640 kbps but with avchigh widevine video

**Comment by cavalia88** [comment]
> The audio stream is not encrypted, only the video. However, Dolby Audio is not supported on Chrome.
> 
> [Dolby Digital Plus is available today in **Microsoft Edge** (Windows 10 and above), Apple Safari 9 on OS X 10.11 (El Capitan and above), and Apple Safari 9 on iOS 9.3 and above.](https://developer.dolby.com/platforms/html5/overview/)

Yes, Netflix audio streams are not encrypted. It's a pity that Dolby Digital Plus / Atmos does not appear to be supported on Chrome browser as yet

I was just wondering on the following:

- Is there a big difference between the AAC 2.0 audio on the Chrome browser and the DDP2.0 audio on the Edge browser? Can normal audience hear the difference?

- Understand that the "AVC High" and "AVC Base" are different H264 encoding profiles. Apart from the difference in profiles, is there a visible difference in picture quality for Netflix streams? Not sure why there appears a preference for the "AVC Base" profile
